pkg/reconciler/pipelinerun/affinity_assistant.go

```diff
@@ -68,6 +68,32 @@ func (c *Reconciler) createAffinityAssistants(ctx context.Context, wb []v1beta1.
 				if err == nil {
 					logger.Infof("Created StatefulSet %s in namespace %s", affinityAssistantName, namespace)
 				}
+			case err == nil && a != nil && a.Status.ReadyReplicas == 1:
+				// the node which hosts the affinity assistant pod is unschedulable or cordoned for some reason,
```

Comment:
The placement of this comment is a bit confusing, since the code having reached this block doesn't necessarily mean the node is unschedulable. Can you reword to state that we need to check whether the node is unschedulable, or move this comment to what's currently line 89? It would also be helpful to link https://github.com/tektoncd/pipeline/issues/6586 in the comment